File: ARCHISOFT_topoalign/frmResetTopo.vb
Public Class frmResetTopo

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Me.DialogResult = System.Windows.Forms.DialogResult.OK
        Me.Close()
    End Sub

    Private Sub cboTopoSource_SelectedIndexChanged(sender As Object, e As EventArgs) Handles cboTopoSource.SelectedIndexChanged, cboTopoTarget.SelectedIndexChanged
        If Me.cboTopoSource.Items.Count > 0 And Me.cboTopoTarget.Items.Count > 0 Then
            Me.Button1.Enabled = True
        End If
    End Sub

End Class

File: ARCHISOFT_topoalign/frmResetTopo.cs
using System;

namespace ARCHISOFT_topoalign
{
    public partial class frmResetTopo
    {
        public frmResetTopo()
        {
            InitializeComponent();
            _Button1.Name = "Button1";
            _cboTopoSource.Name = "cboTopoSource";
            _cboTopoTarget.Name = "cboTopoTarget";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            DialogResult = System.Windows.Forms.DialogResult.OK;
            Close();
        }

        private void cboTopoSource_SelectedIndexChanged(object sender, EventArgs e)
        {
            if (cboTopoSource.Items.Count > 0 & cboTopoTarget.Items.Count > 0)
            {
                Button1.Enabled = true;
            }
        }
    }
}